content/docs/operating/security.md

```diff
@@ -106,7 +106,7 @@ If using a client-library-provided HTTP handler, it should not be possible for
 malicious requests that reach that handler to cause issues beyond those
 resulting from additional load and failed scrapes.
 
-## Authentication/Authorisation/Encryption
+## Authentication, Authorisation, and Encryption
```

Comment:
Hmm, aren't we meant to be using US spellings? I must have let this one slip though.

Reply:
Don't care which as long as we are consistent; changed.